integration/ramdisk/ramdisk.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+cidir=$(dirname "$0")
+
+source "${cidir}/../../lib/common.bash"
+
+set -e
+
+# Environment variables
+IMAGE="${IMAGE:-busybox}"
+CONTAINER_NAME="${CONTAINER_NAME:-test}"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-sleep 30}"
+
+# Set the runtime if not set already
+RUNTIME="${RUNTIME:-kata-runtime}"
+
+# File to save ramdisk results
+TMP_FILE=$(mktemp ramdisk.XXXXXXXXXX)
+
+remove_tmp_file() {
+	rm -rf $TMP_FILE
+}
+
+trap remove_tmp_file EXIT
+
+setup() {
+	clean_env
+
+	# Check that processes are not running
+	check_processes
+
+	# Stop docker
+	sudo -E systemctl stop docker
+}
+
+test_ramdisk() {
+	# Enable ramdisk with dockerd
+	(sudo -E DOCKER_RAMDISK=true dockerd -D --add-runtime ${RUNTIME}=${RUNTIME_PATH} 2>/dev/null) &
+
+	# Run container
+	docker run --rm -d --name=${CONTAINER_NAME} --runtime=${RUNTIME} ${IMAGE} sh -c ${PAYLOAD_ARGS}
+
+	# Extract the logs of kata-runtime
+	sudo journalctl -t ${RUNTIME} > ${TMP_FILE}
+
+	# Verify that --no-pivot flag is at the log
+	check_pivot=$(grep -E -o "\-\-\no\-\pivot" ${TMP_FILE} | wc -l)
+	if [ $check_pivot -eq 0 ]; then
+		echo >&2 "ERROR: --no-pivot flag was not found"
+		exit 1
+	fi
+}
+
+teardown() {
+	dockerd_id=$(pgrep -f dockerd)
+	for i in "${dockerd_id[@]}"; do
+		sudo kill -9 $i
```

Comment:
you might just be able to do a `pkill` here?

Reply:
`pkill` did not work for me, I tried to do `pkill` of the `shim` for example and the process was still there, `kill -9` worked

Reply:
@GabyCT did you try with `pkill --signal 9` ?

Reply:
@marcov yes, I tried that and at least in my environment for example the `shim` was still there, that is why I am using the `kill`